recipes/image_prep.rb

```diff
@@ -16,3 +16,10 @@
   notifies :stop, 'service[splunk]', :before
   notifies :delete, 'file[splunk-marker]', :before
 end
+
+execute 'clean eventdata' do
+  command "#{node['splunk']['cmd']} clean eventdata -f"
+  user node['splunk']['user']
+  group node['splunk']['group']
+  only_if { node['splunk']['node_type'] != :forwarder }
```

Comment:
I wonder if we should be using the nodetype or if we should use the package base name... as the heavy forwarder recipe also sets node_type forwarder: https://github.com/cerner/cerner_splunk/blob/2.14.0/recipes/heavy_forwarder.rb#L9